sdk/src/node/services/da.rs

```diff
@@ -52,8 +52,8 @@ pub trait DaService {
         <Self::Spec as DaSpec>::CompletenessProof,
     );
 
-    // TODO: add a send_transaction method
-    // fn send_transaction(tx: Self::Transaction, sender: Self::Address)
+    /// A method to send a transaction. Takes in some arbitrary transaction payload and the address of the sender
+    fn send_transaction(&self, tx: <Self::Spec as DaSpec>::BlobTransaction, sender: <Self::Spec as DaSpec>::Address);
```

Comment:
Ah sorry, this comment is a bit misleading. The API for sending transactions is still somewhat unclear, because someone has to *sign* the transaction before it can be broadcast - which also means that we probably can't specify what address we want it to be signed with. 

My current feeling is that this method should probably just take a single `&[u8]` argument, and forward it to some external service which handles signing and broadcasting - but we might want to handle some more complicated orchestration instead (for example - maybe we'd add a separate method for "sign transaction" which invokes a wallet API, and then modify `send_transaction` to accept the pre-signed transaction as its argument).